Makefile

```diff
@@ -18,4 +18,4 @@ black:
 
 bandit:
 	$(info ---------- BANDIT ----------)
-	poetry run bandit -c "pyproject.toml" --recursive .
+	poetry run bandit -c "pyproject.toml" --exclude */test_*.py --recursive .
```

Comment:
Move exclusion to pyproject.toml tool.bandit section

Reply:
Addressed in ead2a00157f68d24c543bc986fd0bf5e6aed7b7f